modules/microshift-building-apps-rpms.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-building-apps-rpms_{context}"]
+= Building the RPM package for the application manifests
+
+To building your own RPMs, you must create a `.spec` file that adds the application manifests to the RPM package. The following is an example procedure. As long as your RPMs are accessible to Image Builder, you can use the method you prefer.
```

Comment:
I would remove this section completely. I'm not sure we should be telling users how to build RPMs, etc.

Reply:
@DanielFroehlich and @dhellmann Could you weigh in? If this is how users embed apps, it seems like the correct thing to have a tutorial on this from a Docs perspective. If we have this content in the RHEL docs, I haven't found it. We do include the following link as well: https://www.redhat.com/sysadmin/create-rpm-package[How to create a Linux RPM package].

Reply:
we really need to decide on the style of this. It's either a rather hands on tutorial (blog style), then we should also keep the rpmbuild section to. To be honest, I am struggly a lot with only high level scetches of what needs to be done. E.g. the rhel/edge docs are that, and it is really hard and takes a long time of trial and error to puzzle all the pieces together. Esp. as k8s app developers / admins are usually NOT familiar with building rpms. If we remove this, we could also remove all the other hands on steps and just keep the highlevel "outline procedure" we currently have in the beginning. I sugget we keep it to match that tutorial style.

Reply:
I prefer also to keep the information, then remove or update depending on the feedback of customers. We see a lot of doc bugs such as "I don't know why I'm doing this" and "I don't know what to do," so more is more in many cases. When we stick to outlines in docs, we do lose that conceptual connective tissue of "why."